tt_metal/include/compute_kernel_api.h

```diff
@@ -457,6 +457,14 @@ ALWI void power_tile_init() {
     MATH(( llk_math_eltwise_unary_sfpu_power_init<APPROX>() ));
 }
 
+ALWI void max_tile(uint32_t idst) {
```

Comment:
It would be good to add a description of this API. :)

Reply:
applied the feedback